User_Story#1/SortingApp/BusinessLayer/Utilities/Parser/ArrayParser.cs

```diff
@@ -35,6 +35,9 @@ public ArrayParser()
         /// <param name="content">Content of the file.</param>
         public T[] ConvertData(string content)
         {
+            if (string.IsNullOrWhiteSpace(content))
+                throw new ArgumentNullException(nameof(content));
+
             try
```

Comment:
Why do we have a try-catch here?

Reply:
Because TypeParser inside of block "try" also could throw Exceptions.